cuda_bindings/cuda/bindings/_path_finder/README.md

```diff
@@ -0,0 +1,61 @@
+# `cuda.bindings.path_finder` Module
+
+## Public API (Work in Progress)
+
+Currently exposes two primary interfaces:
+
+```
+cuda.bindings.path_finder._SUPPORTED_LIBNAMES  # ('nvJitLink', 'nvrtc', 'nvvm')
+cuda.bindings.path_finder._load_nvidia_dynamic_library(libname: str) -> LoadedDL
+```
+
+**Note:**
+These APIs are prefixed with an underscore because they are considered
+experimental while undergoing active development, although already
+reasonably well-tested through CI pipelines.
+
+## Library Loading Search Priority
+
+The `load_nvidia_dynamic_library()` function implements a hierarchical search
+strategy for locating NVIDIA shared libraries:
+
```

Comment:
Also, if someone has already loaded libraries via some other mechanism I think we end up finding those before going down any of these other search paths?

Reply:
yes this should be documented

Reply:
I added commit 1d4420b32ef73905067394676e4ed542d69e2422

(sorry that I forgot to add this originally)